systemvm/debian/opt/cloud/bin/cs/CsAddress.py

```diff
@@ -476,7 +476,12 @@ def fw_vpcrouter(self):
             self.fw.append(["", "front", "-A NETWORK_STATS_%s -o %s -s %s" %
                             ("eth1", "eth1", guestNetworkCidr)])
 
-            self.fw.append(["nat", "front",
+            if self.address["source_nat"]:
```

Comment:
@ustcweizhou @rafaelweingartner @GabrielBrascher can you help review and test this? /cc @PaulAngus @DaanHoogland

Reply:
@rhtyd See my reply on the Issue #2680